server/chat-commands/moderation.ts

```diff
@@ -1425,12 +1429,12 @@ export const commands: ChatCommands = {
 			if (!Chat.forceRenames.has(targetUser.id)) Chat.forceRenames.set(targetUser.id, 0);
 		}
 
-		if (room.roomid !== 'staff') this.privateModAction(`(${targetUser.name} ${forceRenameMessage})`);
-		const roomMessage = room.roomid !== 'staff' ? `«<a href="/${room.roomid}" target="_blank">${room.roomid}</a>» ` : '';
+		this.privateModAction(`(${targetUser.name} ${forceRenameMessage})`);
 		const rankMessage = targetUser.getAccountStatusString();
 		Rooms.global.notifyRooms(
 			['staff'],
-			`|html|${roomMessage}` + Utils.html`<span class="username">${targetUser.name}</span> ${rankMessage} ${forceRenameMessage}`
+			`|html|${room ? `<<${room.roomid}>>` : `<PM:${this.pmTarget}>`}` +
+			Utils.html`<span class="username">${targetUser.name}</span> ${rankMessage} ${forceRenameMessage}`
```

Comment:
This is `|html|`, `<<roomid>>` doesn't work in `|html|`. That's why the old code generated the link directly.

Reply:
```suggestion
			`|html|${roomMessage}` + Utils.html`<span class="username">${targetUser.name}</span> ${rankMessage} ${forceRenameMessage}`
```